UselessMine.cpp

```diff
@@ -338,15 +354,15 @@ void UselessMine::Event (bz_EventData *eventData)
 
                             // Get a random defusal message
                             std::string defusalMessage = defusalMessages.at(randomNumber);
-                            bz_sendTextMessage(BZ_SERVER, BZ_ALLUSERS, formatDefusalMessage(defusalMessage, bz_getPlayerCallsign(mine.defuserID), owner).c_str());
+                            bz_sendTextMessage(BZ_SERVER, BZ_ALLUSERS, formatMineMessage(defusalMessage, bz_getPlayerCallsign(mine.defuserID), owner).c_str());
                         }
                         else
                         {
-                            //let the BD player know that they killed the owner
-                            bz_sendTextMessagef(BZ_SERVER, playerID, "You defused %s's mine", owner);
+                            // Let the BD player know that they killed the owner
+                            bz_sendTextMessagef(BZ_SERVER, victim, "You defused %s's mine", owner);
```

Comment:
`victim` here is a callsign (string), not a player slot (int). Second parameter of `bz_sendTextMessagef` needs to be a player slot